app/src/main/java/mozilla/lockbox/extensions/Observable+.kt

```diff
@@ -31,6 +32,10 @@ fun <T : Any> Observable<T>.debug(): Observable<T> {
         }
 }
 
+fun <T : Any, U : Optional<T>> Observable<U>.filterNull(): Observable<T> {
+    return this.filter { it.value != null }.map { it.value }
+}
```

Comment:
Ug. I hate `filter`. Are we filtering out the things that matches, or only letting the thing that matches through?

I can't decide if this should be `filterNull` or `filterNotNull`.